db/seeds.rb

```diff
@@ -7,3 +7,61 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+team_type = TeamType.find_or_create_by!(name: "Test Team Type", icon: "address-book")
+event_type = EventType.find_or_create_by!(name: "Test Event Type")
+badge_type = BadgeType.find_or_create_by!(name: "Test Badge Type")
+users = []
+persons = []
+
+8.times do |time|
+  users << User.find_or_create_by!(email: "test#{time}@gmail.com") do |user|
```

Comment:
Creating users in this breaks the "the first user that logs in is automatically made an admin" bit if you seed your DB before you log in for the first time - I think it would be fine to not make any users in this.

Reply:
Does a person type not need a corresponding user type?

Reply:
You don't need to have a user attached to a person.